tests/pytest/test_multi.py

```diff
@@ -978,139 +978,316 @@ def testErrorMessage(env):
     res = r.execute_command('JSON.GET', 'doc1', '$')
     r.assertEqual([types_data], json.loads(res))
 
+    r.assertEqual(r.execute_command('HSET', 'hash_key', 'a', '1', 'b', '2'), 2)
```

Comment:
```suggestion
    r.expect('HSET', 'hash_key', 'a', '1', 'b', '2').equal(2)
```

Reply:
Keeping `execute_command` to handle cluster connection